### PR TITLE
Add Departure Mono

### PR DESCRIPTION
**Pull request by carterworks**
https://departuremono.com/

> Departure Mono is a monospaced pixel font inspired by the constraints of early command-line and graphical user interfaces, the tiny pixel fonts of the late 90s/early 00s, and sci-fi concepts from film and television.

By [Helena Zhang](https://www.helenazhang.com/)

image

### CODE REVIEW

**Comment by braver** [comment]
Oh wow, and that website is _fantastic_ too!

File: fonts.json [hunk 1]
```diff
@@ -544,6 +544,18 @@
...
         "zerostyle": "dotted"
     },
+    "departure-mono": {
+        "author": "Helena Zhang",
+        "description": "Departure Mono is a monospaced pixel font inspired by the constraints of early command-line and graphical user interfaces, the tiny pixel fonts of the late 90s/early 00s, and sci-fi concepts from film and television.",
+        "license": "SIL OFL",
+        "ligatures": false,
+        "name": "Departure Mono",
+        "rendering": "vector",
+        "style": "sans",
```
Comment (braver) [inline fonts.json:554]:
```suggestion
        "style": "sans",
        "variants": [
            "regular"
        ],
```

**Review by braver — CHANGES_REQUESTED** [review]
Thanks! I think we could add the variants info, but otherwise it looks good to 🚀

**Comment by braver** [comment]
Preferably also check in the LICENSE file along with the font files.

(By the way, if you don't get around to handling the PR feedback, don't worry I can definitely take it from here)

**Comment by carterworks** [comment]
> Preferably also check in the LICENSE file along with the font files.


Done.

**Review by braver — APPROVED** [review]

**Pull request merged**